api/database/seeders/PoolSeeder.php

```diff
@@ -25,6 +26,7 @@ public function run()
                 ],
                 'key' => 'digital_careers',
                 'user_id' => User::where('email', 'admin@test.com')->first()->id,
+                'team_id' => Team::where('name', 'digital-community-management')->sole()['id'],
```

Comment:
For consistency, can we access the `id` property with small arrow syntax?

```suggestion
                'team_id' => Team::where('name', 'digital-community-management')->sole()->id,
```

Reply:
just mimicked the line above instead https://github.com/GCTC-NTGC/gc-digital-talent/pull/5561/commits/883ecee07fdc65dc74ec0d7acc1625eed579b34e